pkg/daemon/metrics.go

```diff
@@ -416,37 +414,6 @@ func extractPTP4lEventState(output string) (portId int, role ptpPortRole) {
 	return
 }
 
-func addFlagsForMonitor(nodeProfile *ptpv1.PtpProfile, stdoutToSocket bool) {
```

Comment:
Is this still something we want?  If so, it should be added by operator not here.

Reply:
This change might  break cnf-test 
Also - m  required for regular Prometheus metrics (for non-events,)  and  -u to be removed for events to generated S1/S2 clock state.
Unless you add these as part of your webhook validations for ptpconfig and ptpoperatorconfig (foe event flag) updates

Reply:
How can an operator add this flag? are you suggesting modifying ptpconfig CR  by the operator when the event is enabled and disabled?
I am okay with these changes only If we can retain this functionality somewhere else and then remove this. Also please run and functests  test for these changes.

Reply:
Is there any reason we can't just expect a user to provision this correctly?